lib/elixir/lib/keyword.ex

```diff
@@ -34,7 +34,7 @@ defmodule Keyword do
   of a keyword without quotes, Elixir will warn.
 
   Note that when keyword lists are passed as the last argument to a function,
-  if the short-hand syntax is used then the square brackets around the keyword list
+  if the short hand syntax is used then the square brackets around the keyword list
```

Comment:
I think that the [correct spelling is "shorthand"](https://www.merriam-webster.com/dictionary/shorthand)?

```suggestion
  if the shorthand syntax is used then the square brackets around the keyword list
```